### ISSUE TITLE
[4.0.0-beta.16] classList.forEach is not a function error in Safari 9.1

### ISSUE THREAD

**Issue by mos-ie** [ionic-team/ionic#16490]
# Bug Report

**Ionic Info**
Run `ionic info` from a terminal/cmd prompt and paste the output below.

```
Ionic:

   ionic (Ionic CLI)             : 4.4.0 (C:\Users\Matthew\AppData\Roaming\npm\node_modules\ionic)
   Ionic Framework               : @ionic/angular 4.0.0-beta.16
   @angular-devkit/build-angular : 0.10.7
   @angular-devkit/schematics    : 0.7.5
   @angular/cli                  : 7.0.7
   @ionic/angular-toolkit        : 1.2.0

System:

   NodeJS : v10.13.0 (C:\Program Files\nodejs\node.exe)
   npm    : 6.3.0
   OS     : Windows 10
```

**Describe the Bug**
Testing on Safari 9.1 desktop on BrowserStack. When the application loads in the browser a `TypeError: classList.forEach is not a function. (In 'classList.forEach', 'classList.forEach' is undefined) ` error is logged in the console from vendor.js -> @ionic/angular/dist/directives/control-value-accessors/value-accessor.js

**Related Code**
```
function setIonicClasses(element) {
    requestAnimationFrame(function () {
        var classList = element.nativeElement.classList;
        classList.remove('ion-valid', 'ion-touched', 'ion-untouched', 'ion-dirty', 'ion-pristine');
        classList.forEach(function (cls) {
            if (cls.startsWith('ng-')) {
                classList.add("ion-" + cls.substr(3));
            }
        });
    });
}
```

**Comment by manucorporat** [comment]
This is already fixed in master!

**Comment by mos-ie** [comment]
Excuse my ignorance, but does that not mean it should be fixed in 4.0.0-beta.16?
